cmd/aro/main.go

```diff
@@ -55,31 +54,31 @@ func main() {
 	switch strings.ToLower(flag.Arg(0)) {
 	case "dbtoken":
 		checkArgs(1)
-		err = dbtoken(ctx, log)
```

Comment:
For POC, we'll just use entry point POC, so don't build the other components of ARO-RP